### ISSUE TITLE
Spec & déploiement SemApps CdlT - Amorçage !

### ISSUE THREAD

**Issue by GuillaumeAV** [chemins-de-la-transition/plateforme#256]
### Enjeu
Développer et déployer le socle de la plateforme des Chemins de la Transition, avec les concepts et fonctionnalités de base, sans aller dans la finesse ou la complexité, pour ne pas avoir besoin de spécifier, pour ne pas avoir à défaire des trucs parce qu'on aurait mal spécifié. 

Une bonne partie du travail peut-être réalisée via l’ontologie PAIR + le ou les modules qu’il s’agira de concevoir pour répondre aux besoins spécifiques des CdlT

### Mission 1 : Implémenter les classes nécessaires aux CdlT
Les Chemins de la transition se structurent autour de quelques concepts clés, dont certains à créer (en gras) qu'il s'agit d'implémenter :
- Acteurs : Individus et organisations
- Types d’acteurs 
     - **Apprenant / Voyageur / Wwoofer / Etudiant / Compagnon** (Choisir le terme / label - pas problématique)
     - **Hote / Accueillant / enseignant / …** (Choisir le terme / label - pas problématique)
- Lieux 
- **Parcours** (concept pouvant être créé en tant que sous classe de projet)
- **Type de parcours** (Pour le moment - on pourra créer des classes en dur par la suite)
  - **Ecotourisme** (Les termes importent peu ...)
  - **Voyage apprenant** (Les termes importent peu ...)
  - **Woofing**
  - **Compagnonage**
- Compétences
- Thèmes
- Evénement 
- Idée
- Statut
    - En cours de création
    - Opérationnel 
    - Révolu
- Domaines, secteurs, branches .. [Moins prioritaire] 
- Finalité [Moins prioritaire] 
- Enjeu [Moins prioritaire] 

### Mission 2 : Implémenter la classe parcours et ses relations avec le reste de l'ontologie
Les parcours sont le concept clé, ils interagissent avec concepts cités ci-dessus via les relations suivantes : 
- a pour participant : Apprenants
- a pour organisateur : Accueillants
- est composé de : Lieux 
- est de type : Ecotourisme, Voyage apprenant ... 
- produit : Compétences 
- a pour thème : Thème 
- fait l'objet de : Evenement
- découle de : Idée
- a pour statut : Statut
- a pour secteur / branche / domaine : Secteur / Branche / Domaine
- [Moins prioritaire] poursuit : Finalité 
- [Moins prioritaire] a pour enjeu : Enjeu

### Mission 3 (par les PO & designers) 
Designer sur Figma un écran pour voir comment on joue avec la superclasse concept [en haut de ce diagramme](https://raw.githubusercontent.com/assemblee-virtuelle/pair/master/3-Dissemination/Diagrammes-PAIR-Winter-2020/PAIR%20Core%20Subjects-2020-WinterEdition.png) dont les éléments constitutifs (types de.., thèmes, statuts, champs d'activité) peuvent être appréhendés comme des filtres.

### Mission 4 : Implémenter les concepts de la superclasse concept dans l'interface ;) 
Si mission 3 pas satisfaisante, passer à la mission 5. 

### Mission 5 : Spécifier et implémenter Wikidata
Pour gérer ainsi les thèmes et compétences à minima. 

### Mission 6  : Travailler sur l'interopérabilité ...

**Comment by J9rem** [comment]
Point de vigilance : ce sont des spec. fonctionnelles de haut niveau.
Or, le système de documentation Chemins de la Transition est sur G Drive.
Je pense que ce document n'est pas au bon endroit donc.

**Comment by GuillaumeAV** [comment]
Yes t'as raison ! [Voici un gdoc sur grdrive](https://docs.google.com/document/d/1IwUlCyhlw-EYfsn2x2E51WFHWmLCYntXL9mPixkIo5E/edit?usp=sharing) :)
